fuzzers/051-pip-imuxlout-bypalts/generate.tcl

```diff
@@ -26,6 +26,49 @@ route_design
 
 # write_checkpoint -force design.dcp
 
+proc find_dst_pin {tile dst_wire} {
```

Comment:
Please add comments about what this does, inputs and outputs.  Same for find_src_pin.

Reply:
done